### ISSUE TITLE
unknown method blockchain.scripthash.listunspent [String(" when trying to swipe a private key

### ISSUE THREAD

**Issue by gabridome** [romanz/electrs#36]
If I try to swipe a private key into electrum the server stop responding and in the log of the server appears:

````
2018-07-21T17:17:08.388+02:00 - ERROR - [127.0.0.1:33256] connection handling failed: Error: unknown method blockchain.scripthash.listunspent [String("c09711449cd08e347cc70a32f6f4efacf5ef75c1d7d17dc53c84b9ade0af5143")]

2018-07-21T17:17:08.403+02:00 - INFO - [127.0.0.1:33256] shutting down connection
2018-07-21T17:17:08.404+02:00 - INFO - [127.0.0.1:33256] disconnected peer
2018-07-21T17:17:08.958+02:00 - INFO - [127.0.0.1:33308] connected peer
2018-07-21T17:17:09.466+02:00 - ERROR - [127.0.0.1:33308] connection handling failed: Error: unknown method blockchain.scripthash.listunspent [String("c09711449cd08e347cc70a32f6f4efacf5ef75c1d7d17dc53c84b9ade0af5143")]

2018-07-21T17:17:09.466+02:00 - INFO - [127.0.0.1:33308] shutting down connection
2018-07-21T17:17:19.072+02:00 - INFO - [127.0.0.1:33310] connected peer
2018-07-21T17:17:19.548+02:00 - ERROR - [127.0.0.1:33310] connection handling failed: Error: unknown method blockchain.scripthash.listunspent [String("c09711449cd08e347cc70a32f6f4efacf5ef75c1d7d17dc53c84b9ade0af5143")]
````

**Comment by greenaddress** [comment]
this is related to https://github.com/romanz/electrs/issues/25

**Comment by romanz** [comment]
Indeed, this issue will be fixed by resolving #25.
